helpers/cli.js

```diff
@@ -90,35 +90,40 @@ const main = async () => {
 		space: false,
 	});
 
-	let name = process.argv.slice(2)[0],
+	let packageName = process.argv.slice(2)[0],
 		description;
 
-	if (!name) {
+	if (!packageName) {
 		do {
-			name = await qName().catch(e => console.log(chalk.red(e)));
-		} while (!name);
+			packageName = await qName().catch(e => console.log(chalk.red(e)));
+		} while (!packageName);
 	}
 
 	do {
 		description = await qDescription().catch(e => console.log(chalk.red(e)));
 	} while (!description);
 
-	const folderName = name.toLowerCase();
+	const folderName = packageName.toLowerCase();
 
-	name = name
+	const compomentTitle = packageName
+		.split('-')
+		.map(part => part[0].toUpperCase() + part.slice(1))
+		.join(' ');
+
+	const componentName = packageName
```

Comment:
we can just do 
```
const componentName = componentTitle;
```
one less calculation

Reply:
componentName has no spaces ("FormPod"), componentTitle has spaces ("Form Pod")

Reply:
how about

```const componentName = componentTitle.replace(' ', '');```